components/kyma-environment-broker/internal/process/upgrade_kyma/btp_operator_overrides.go

```diff
@@ -31,7 +31,8 @@ func (s *BTPOperatorOverridesStep) Run(operation internal.UpgradeKymaOperation,
 	if clusterID == "" {
 		var err error
 		if clusterID, err = ConfigMapGetter(operation.InstanceDetails.Kubeconfig); err != nil {
-			return s.operationManager.OperationFailed(operation, "failed to create BTP Operator input", err, log)
+			log.Error("failed to create BTP Operator input", err)
+			return operation, 30 * time.Second, nil
```

Comment:
the problem is we do not rerun get kubeconfig here ( as I see)

Reply:
ah, you are right. I was under the wrong impressinon that the whole stage of steps will be requeued. Thanks for pointing it out. I think the logging part of the PR still has value and we can verify whether the `nil` assumption is correct, how frequent it is and then alternatively requeue the `GetKubeconfig` step when the value is this particular buggy kubeconfig.

Reply:
I have added requeue to kubeconfig fetching when kubeconfig has suspiciously small number of characters